beagle/src/main/kotlin/br/com/zup/beagle/android/components/list/ListViewHolder.kt

```diff
@@ -37,12 +37,12 @@ import br.com.zup.beagle.android.widget.core.MultiChildComponent
 import br.com.zup.beagle.android.widget.core.ServerDrivenComponent
 import br.com.zup.beagle.android.widget.core.SingleChildComponent
 import org.json.JSONObject
-import java.util.LinkedList
+import java.util.*
```

Comment:
Wyld card. In the past, people asked me to remove it. I'm not sure if It is still a rule.